### ISSUE TITLE
Issues installing plugin

### ISSUE THREAD

**Issue by crtorreg** [aarpon/qu#2]
Hi, 

I am having the following issues while installing (pip install -e .) the plugin into my computer: 

`INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of qu to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement cucim~=0.19.0; extra == "all" (from monai[all]) (from versions: none)
ERROR: No matching distribution found for cucim~=0.19.0; extra == "all"`

Thank you, 
Cristina

**Comment by aarpon** [comment]
Hi, Cristina

I tried to reinstall Qu from scratch and -- although I realized there was a missing dependency in setup.cfg ([I pushed a fix](https://github.com/aarpon/qu/commit/a8ed81f1fde331fdd9f1fd04c540386132257ebd)) -- I could not reproduce your problem. 

Could we try something? Could you please delete your local `qu` git clone, and then run the following (please notice that I renamed the `napari-env` conda environment to `napari-test` to make sure we try in a fresh environment):

```bash
conda create -y -n napari-test python=3.8
conda activate napari-test
pip install "napari[all]"
git clone https://github.com/aarpon/qu
cd qu
pip install -e .
```
If you get the same issue, could you please send me the output of `conda list`?

**Comment by crtorreg** [comment]
Hi, 

I have tried it again but is still giving me the same error. This is the output of `conda list`. Let me know If I am doing something wrong. 

`
(napari-test) C:\Users\equipo\qu>conda list
# packages in environment at C:\Users\equipo\anaconda3\envs\napari-test:
#
# Name                    Version                   Build  Channel
alabaster                 0.7.12                   pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
asttokens                 2.0.5                    pypi_0    pypi
attrs                     21.4.0                   pypi_0    pypi
babel                     2.9.1                    pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
black                     21.12b0                  pypi_0    pypi
ca-certificates           2021.10.26           haa95532_2
cachey                    0.2.1                    pypi_0    pypi
certifi                   2021.10.8        py38haa95532_2
charset-normalizer        2.0.10                   pypi_0    pypi
click                     8.0.3                    pypi_0    pypi
cloudpickle               2.0.0                    pypi_0    pypi
colorama                  0.4.4                    pypi_0    pypi
dask                      2022.1.0                 pypi_0    pypi
debugpy                   1.5.1                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
docstring-parser          0.13                     pypi_0    pypi
docutils                  0.17.1                   pypi_0    pypi
entrypoints               0.3                      pypi_0    pypi
executing                 0.8.2                    pypi_0    pypi
freetype-py               2.2.0                    pypi_0    pypi
fsspec                    2022.1.0                 pypi_0    pypi
git                       2.34.1               haa95532_0
heapdict                  1.0.1                    pypi_0    pypi
hsluv                     5.0.2                    pypi_0    pypi
idna                      3.3                      pypi_0    pypi
imageio                   2.14.1                   pypi_0    pypi
imagesize                 1.3.0                    pypi_0    pypi
importlib-metadata        4.10.1                   pypi_0    pypi
importlib-resources       5.4.0                    pypi_0    pypi
intervaltree              3.1.0                    pypi_0    pypi
ipykernel                 6.7.0                    pypi_0    pypi
ipython                   8.0.1                    pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
jedi                      0.18.1                   pypi_0    pypi
jinja2                    3.0.3                    pypi_0    pypi
jsonschema                4.4.0                    pypi_0    pypi
jupyter-client            7.1.2                    pypi_0    pypi
jupyter-core              4.9.1                    pypi_0    pypi
kiwisolver                1.3.2                    pypi_0    pypi
locket                    0.2.1                    pypi_0    pypi
magicgui                  0.3.4                    pypi_0    pypi
markupsafe                2.0.1                    pypi_0    pypi
matplotlib-inline         0.1.3                    pypi_0    pypi
mypy-extensions           0.4.3                    pypi_0    pypi
napari                    0.4.13                   pypi_0    pypi
napari-console            0.0.4                    pypi_0    pypi
napari-plugin-engine      0.2.0                    pypi_0    pypi
napari-svg                0.1.6                    pypi_0    pypi
nest-asyncio              1.5.4                    pypi_0    pypi
networkx                  2.6.3                    pypi_0    pypi
npe2                      0.1.1                    pypi_0    pypi
numpy                     1.22.1                   pypi_0    pypi
numpydoc                  1.2                      pypi_0    pypi
openssl                   1.1.1m               h2bbff1b_0
packaging                 21.3                     pypi_0    pypi
pandas                    1.4.0                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
partd                     1.2.0                    pypi_0    pypi
pathspec                  0.9.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    9.0.0                    pypi_0    pypi
pint                      0.18                     pypi_0    pypi
pip                       21.2.2           py38haa95532_0
platformdirs              2.4.1                    pypi_0    pypi
pooch                     1.6.0                    pypi_0    pypi
prompt-toolkit            3.0.24                   pypi_0    pypi
psutil                    5.9.0                    pypi_0    pypi
psygnal                   0.2.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pydantic                  1.9.0                    pypi_0    pypi
pygments                  2.11.2                   pypi_0    pypi
pyopengl                  3.1.5                    pypi_0    pypi
pyparsing                 3.0.7                    pypi_0    pypi
pyqt5                     5.15.6                   pypi_0    pypi
pyqt5-qt5                 5.15.2                   pypi_0    pypi
pyqt5-sip                 12.9.0                   pypi_0    pypi
pyrsistent                0.18.1                   pypi_0    pypi
python                    3.8.12               h6244533_0
python-dateutil           2.8.2                    pypi_0    pypi
pytomlpp                  1.0.10                   pypi_0    pypi
pytz                      2021.3                   pypi_0    pypi
pywavelets                1.2.0                    pypi_0    pypi
pywin32                   303                      pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     22.3.0                   pypi_0    pypi
qtconsole                 5.2.2                    pypi_0    pypi
qtpy                      2.0.0                    pypi_0    pypi
requests                  2.27.1                   pypi_0    pypi
scikit-image              0.19.1                   pypi_0    pypi
scipy                     1.7.3                    pypi_0    pypi
setuptools                58.0.4           py38haa95532_0
six                       1.16.0                   pypi_0    pypi
snowballstemmer           2.2.0                    pypi_0    pypi
sortedcontainers          2.4.0                    pypi_0    pypi
sphinx                    4.4.0                    pypi_0    pypi
sphinxcontrib-applehelp   1.0.2                    pypi_0    pypi
sphinxcontrib-devhelp     1.0.2                    pypi_0    pypi
sphinxcontrib-htmlhelp    2.0.0                    pypi_0    pypi
sphinxcontrib-jsmath      1.0.1                    pypi_0    pypi
sphinxcontrib-qthelp      1.0.3                    pypi_0    pypi
sphinxcontrib-serializinghtml 1.1.5                    pypi_0    pypi
sqlite                    3.37.0               h2bbff1b_0
stack-data                0.1.4                    pypi_0    pypi
superqt                   0.2.5.post1              pypi_0    pypi
tifffile                  2021.11.2                pypi_0    pypi
tomli                     1.2.3                    pypi_0    pypi
toolz                     0.11.2                   pypi_0    pypi
tornado                   6.1                      pypi_0    pypi
tqdm                      4.62.3                   pypi_0    pypi
traitlets                 5.1.1                    pypi_0    pypi
typer                     0.4.0                    pypi_0    pypi
typing-extensions         4.0.1                    pypi_0    pypi
urllib3                   1.26.8                   pypi_0    pypi
vc                        14.2                 h21ff451_1
vispy                     0.9.4                    pypi_0    pypi
vs2015_runtime            14.27.29016          h5e58377_2
wcwidth                   0.2.5                    pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0
wincertstore              0.2              py38haa95532_2
wrapt                     1.13.3                   pypi_0    pypi
zipp                      3.7.0                    pypi_0    pypi
`

Thank you a lot!

**Comment by aarpon** [comment]
Can you run `conda --version` and in case the version is lower than `conda 4.11.0` run the following:

```bash
conda deactivate        # Make sure you are in the base environment
conda update conda
```
Then try again: you can use python 3.9 instead of 3.8.

**Comment by crtorreg** [comment]
Hi, 

I have just avoided the required version of monai in the setup file to the following: `monai[all]`. Because the problem was with the specified version of monai, now it works properly but I do not know if it will affect any future step.

**Comment by aarpon** [comment]
No, sorry. This will currently not work. See [issue 3](https://github.com/aarpon/qu/issues/3). What happens if you do this instead?

```bash
pip install monai==0.5.3
```
